parsers/JP-KY.py

```diff
@@ -40,81 +41,85 @@ def fetch_production(
         "storage": {},
         "source": "www.kyuden.co.jp",
     }
+
     # url for consumption and solar
     url = "https://www.kyuden.co.jp/td_power_usages/pc.html"
     r = get(url)
     r.encoding = "utf-8"
     html = r.text
     soup = BeautifulSoup(html, "lxml")
-    # get hours, minutes
-    ts = soup.find("p", class_="puProgressNow__time").get_text()
-    hours = int(re.findall(r"[\d]+(?=時)", ts)[0])
-    minutes = int(re.findall(r"(?<=時)[\d]+(?=分)", ts)[0])
+
     # get date
-    ds = soup.find("div", class_="puChangeGraph")
-    date = re.findall(r"(?<=chart/chart)[\d]+(?=.gif)", str(ds))[0]
+    date_div = soup.find("div", class_="puChangeGraph")
+    date_str = re.findall(r"(?<=chart/chart)[\d]+(?=.gif)", str(date_div))[0]
+    year_str = date_str[:4]
+    month_str = date_str[4:6]
+    day_str = date_str[6:]
```

Comment:
Why do we need to split them up into all these strings?
Cant we just parse it as a date and then use that?

Reply:
Ah, that's correct. I was influenced by the original code a bit.

I'll adjust that date parsing soon.

Reply:
Yeah I'm aware and I want to update but some of the libs we use in the backend + the version of pandas we use (I think) don't support Python 3.11 yet and we want to keep them in sync.

It's really annoying though 😅

Reply:
That makes sense😅 I'm OK with 3.10 for now. Hopefully these dependencies will gradually adopt to newer versions.